kafka_consumer/tests/common.py

```diff
@@ -104,7 +117,7 @@ def assert_check_kafka(aggregator, consumer_groups):
                 for mname in CONSUMER_METRICS:
                     aggregator.assert_metric(
                         mname,
-                        tags=tags + [f"consumer_group:{name}"],
+                        tags=tags + [f"consumer_group:{name}"] + [f"consumer_group_state:{consumer_group_state}"],
```

Comment:
My 2 cents: I would lean towards simplifying the integration tests.

So instead of us collecting the `consumer_group_state`, I'd simply ignore that tag for integration tests. We're already checking it in unit tests.

"But how do I get the tests to pass?" you might ask? Very reasonable. Here's how.

You replace the one  `aggregator.assert_metric` call with this for-loop:
```python
for t in tags + [f"consumer_group:{name}"]:
    aggregator.assert_has_tag(mname, t)
```
Taking advantage of [this aggregator method](https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.stubs.aggregator.AggregatorStub.assert_metric_has_tag).

Reply:
From a consistency point of a view, we are testing the other tags (consumer_group name, topic, partition and custom tags)  in the integration test, if we skip the consumer_group_state tag and then a change happens with the confluent kafka library on how this information(consumer_group_state) is exposed, wouldn't it be missed because we removed it from the integration test?

Reply:
What if you use [this method](https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.stubs.aggregator.AggregatorStub.assert_metric_has_tag_prefix) and assert about `consumer_group_state` without including the state part which could vary?